### PR TITLE
Added styling for about page and rendered cards for the maintainers. …

### LINKED ISSUE #189: Add maintainer cards to about page
In `about.tsx` remove the WorkInProgress component and replace it with an h2 that says Current Maintainers. 

Import the MaintainerCard into the file and remove the WorkInProgress import.

Directly beneath the h2 map over the maintainerData and pass each object into the maintainerCard component.

Verify that 2 cards show on the screen with names, descriptions and github links (images will be added later).

This issue is for @sano2019

### PR DESCRIPTION
**Pull request by sano2019**
## Describe your changes
Added the cards for maintainers as requested.

## Link to issue

Closes #189 

## Checklist before requesting a review

- [X] I have performed a self-review of my code.
- [X] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [X] I ran the app and tested it locally to verify that it works as expected.
- [x] I have checked my code with an automatic accessibility tool such as Axe Dev Tools or Wave 
      and it shows no errors.
## Additional Information (Optional)
Any additional information that you want to give us.

<a href="https://gitpod.io/#https://github.com/AccessibleForAll/AccessibleWebDev/pull/190"></a>

### CODE REVIEW

File: pages/about.tsx [hunk 1]
```diff
@@ -46,7 +53,8 @@ export const getStaticProps: GetStaticProps = async (context) => {
 		{
 			image: "",
 			fullName: "Cristian Toffanin",
-			description: "",
+			description:
+				"Doggo ipsum many pats long doggo shooberino shoober aqua doggo, pupperino mlem h*ck. Woofer much ruin diet heckin good boys heckin adorable doggo, super chub yapper.",
```
Comment (ctoffanin) [inline pages/about.tsx:57]: adorable doggo 🤣

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Thanks Sander for helping :D Cristian approves of the Doggo Ipsum 🤣 

Just a couple of things that need adjusting and can you please revert the changes to the yarn.lock file, that doesn't need to be in the PR since you didn't install any new packages.

**Comment by sano2019** [comment]
Updated the changes as requested :)

**Review by EmmaDawsonDev — APPROVED** [review]
Awesome, thank you :D

**Pull request merged**